Dockerfile

```diff
@@ -6,6 +6,6 @@ COPY . /app
 
 RUN chmod -R 755 /app/*
 
-# # Proceed to install the requirements...do
-RUN cd /app && apt-get update && \
+# Proceed to install the requirements...do
+RUN cd /app && apt-get --allow-releaseinfo-change update && \
```

Comment:
I think this will work, but it will become a problem down the line. It defeats a reasonable and helpful restriction of the package manager. I think a more elegant solution is to get the base image of this image in sync with this repo's (correct and up-to-date) idea of what is `stable` and what is `oldStable`. We could also tie ourselves not to these floating `stable` & `oldStable` and use the named releases which never change.

Reply:
You're totally right. The base image is what should be updated, and this can be left as is then.

Reply:
Could one of y'all change this for me? I'm so lost 😅  @frankhereford @chiaberry

Reply:
I'll take a stab at it @Charlie-Henry ! I'll keep you updated